src/lib/udmp-parser.h

```diff
@@ -682,18 +723,36 @@ struct Module_t {
   uint32_t TimeDateStamp = 0;
   std::string ModuleName;
   dmp::FixedFileInfo_t VersionInfo;
-  const void *CvRecord = nullptr;
+  std::unique_ptr<uint8_t[]> CvRecord{};
   uint32_t CvRecordSize = 0;
-  const void *MiscRecord = nullptr;
+  std::unique_ptr<uint8_t[]> MiscRecord{};
   uint32_t MiscRecordSize = 0;
 
   Module_t(const dmp::ModuleEntry_t &M, const std::string &Name,
            const void *CvRecord_, const void *MiscRecord_)
       : BaseOfImage(M.BaseOfImage), SizeOfImage(M.SizeOfImage),
         CheckSum(M.CheckSum), TimeDateStamp(M.TimeDateStamp), ModuleName(Name),
-        VersionInfo(M.VersionInfo), CvRecord(CvRecord_),
-        CvRecordSize(M.CvRecord.DataSize), MiscRecord(MiscRecord_),
-        MiscRecordSize(M.MiscRecord.DataSize) {}
+        VersionInfo(M.VersionInfo), CvRecordSize(M.CvRecord.DataSize),
+        MiscRecordSize(M.MiscRecord.DataSize) {
+    //
+    // Take ownership of `CvRecord` and `MiscRecord`
+    //
+    CvRecord = std::make_unique<uint8_t[]>(CvRecordSize);
+    ::memcpy(CvRecord.get(), CvRecord_, CvRecordSize);
```

Comment:
I think you used `memcpy` directly earlier; can we use either one or the other everywhere for consistency?